src/status_im/ui/screens/wallet/subs.cljs

```diff
@@ -111,6 +111,11 @@
                   (fn [[balance visible-assets]]
                     (map #(assoc % :amount (get balance (:symbol %))) visible-assets)))
 
+(re-frame/reg-sub :wallet/transferrable-assets-with-amount
+                  :<- [:wallet/visible-assets-with-amount]
+                  (fn [all-assets]
+                    (filter #(not (:nft? %)) all-assets)))
```

Comment:
shorter version would be:
```clj
(remove :nft? all-assets)
```

Reply:
Indeed, but it doesn't matter too much now. I'm going to post a separate refactor PR to add token `:type` field with values `:erc20`, `:erc721` etc.